api/pkg/backend/service.go

```diff
@@ -447,3 +446,133 @@ func (s *APIService) listStorageType(ctx context.Context, request *restful.Reque
 	storageTypes, err := s.backendClient.ListType(ctx, listTypeRequest)
 	return storageTypes, err
 }
+
+func (s *APIService) CreateTier(request *restful.Request, response *restful.Response) {
+	log.Info("Received request for creating tier.")
+	tier := &backend.Tier{}
+	err := request.ReadEntity(&tier)
+	if err != nil {
+		log.Errorf("failed to read request body: %v\n", err)
+		response.WriteError(http.StatusInternalServerError, err)
+		return
+	}
+	ctx := common.InitCtxWithAuthInfo(request)
+	actx := request.Attribute(c.KContext).(*c.Context)
+	tier.TenantId = actx.TenantId
+	tier.UserId = actx.UserId
+
+	res, err := s.backendClient.CreateTier(ctx, &backend.CreateTierRequest{Tier: tier})
```

Comment:
As mentioned in some other comment, it should handle list of backends in a single tier. Please check logic again.

Reply:
done

Reply:
What happens if, multiple backends are added, and 1st backnd itself doesn't exist. It won't check for other backends,

Reply:
added